packages/api/src/index.ts

```diff
@@ -84,7 +84,7 @@ class API extends LCDClient {
     const fee = opts.fee || await (async (): Promise<IFee> => {
       const gas = await this.simulate(msgs, account.address, chain_id, account_number, sequence)
       return {
-        amount: [{ denom: 'atto', amount: gas }], // amount = gas because minGasPrice=1.0
+        amount: [],
```

Comment:
Tests are failing but it seems that setting the amount to 0 solves the issue. 
```suggestion
        amount: [{ denom: 'atto', amount: '0' }], // amount = gas because minGasPrice=0
```

Reply:
did you had a look at the reason of that to make sure that we will not have issues somewhere else and that it is the correct way to do?